pibooth_picture_template.py

```diff
@@ -199,9 +199,6 @@ def parse_captures(self, mxgraph_node):
         """
         captures = [cell for cell in mxgraph_node.iter('mxCell')
                     if cell.get('vertex') == "1" and not cell.get('style').startswith('text;')]
-        if len(captures) > 4 or len(captures) < 1:
-            raise TemplateParserError("Too many captures positions ({}) in template '{}' (1 to 4 expected)".format(
-                len(captures), mxgraph_node.get('name')))
```

Comment:
I don't understand why you need to remove this protection

Reply:
Because we can expect unlimited number of captures, as you can duplicate them as much as you want.